script/input/1/mainnet-production.json

```diff
@@ -1,10 +1,14 @@
 {
+    "admin": "0x3300f198988e4C9C63F75dF86De36421f06af8c4",
+    "allocatorVault": "0x691a6c29e9e96dd897718305427Ad5D534db16BA",
+    "almProxy": "0x1601843c5E9bC251A3272907010AFa41Fa18347E",
     "cctpTokenMessenger": "0xBd3fa81B58Ba92a82136038B25aDec7066af3155",
-    "safe": "0x611C7c37F296240c2fF5a92f0B4a398B01B237c4",
-    "usdc": "0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48",
     "dai": "0x6B175474E89094C44Da98b954EedeAC495271d0F",
-    "usds": "0xdC035D45d973E3EC169d2276DDab16f1e407384F",
-    "susds": "0xa3931d71877C0E7a3148CB7Eb4463524FEc27fbD",
     "daiUsds": "0x3225737a9Bbb6473CB4a45b7244ACa2BeFdB276A",
-    "psm": "0xf6e72Db5454dd049d0788e411b06CfAF16853042"
+    "psm": "0xf6e72Db5454dd049d0788e411b06CfAF16853042",
+    "rateLimits": "0x7A5FD5cf045e010e62147F065cEAe59e5344b188",
+    "safe": "0x611C7c37F296240c2fF5a92f0B4a398B01B237c4",
```

Comment:
Production safe is `0x8a25A24EDE9482C4Fc0738F99611BE58F1c839AB`

Reply:
Updated